src/main/resources/db_scripts/db_script_v0.sql

```diff
@@ -0,0 +1,45 @@
+create table "workflow" (
+  "name" VARCHAR(45) NOT NULL UNIQUE,
+  "is_active" BOOLEAN NOT NULL,
+  "created" TIMESTAMP NOT NULL,
+  "updated" TIMESTAMP,
+  "id" BIGSERIAL NOT NULL PRIMARY KEY
```

Comment:
`BIGSERIAL` should include `NOT NULL`, does it?

Reply:
I am not sure. If yes, I will fix that but it has to be in new pr. Point of this pr is to set convention for DB scripts and delta scripts.